plonky2/Cargo.toml

```
@@ -23,7 +23,7 @@ ahash = { workspace = true }
 anyhow = { workspace = true }
 hashbrown = { workspace = true }
 itertools = { workspace = true }
-keccak-hash = { version = "0.10.0", default-features = false }
+tiny-keccak = { version = "2.0.0", features = ["keccak"] }
```

```suggestion
tiny-keccak = { version = "2.0", features = ["keccak"] }
```

No need to lock down the version so much.  That's what we have `Cargo.lock` files for.